src/auth-service/routes/api-v1.js

```diff
@@ -118,26 +118,28 @@ router.post(
 router.post(
   "/firebase/lookup",
   oneOf([
-    [
-      body("email")
-        .exists()
-        .withMessage("the email must be provided")
-        .bail()
-        .notEmpty()
-        .withMessage("the email must not be empty if provided")
-        .bail()
-        .isEmail()
-        .withMessage("this is not a valid email address"),
-      body("phoneNumber")
-        .exists()
-        .withMessage("the phoneNumber must be provided")
-        .bail()
-        .notEmpty()
-        .withMessage("the phoneNumber must not be empty if provided")
-        .bail()
-        .isMobilePhone()
-        .withMessage("the phoneNumber must be valid"),
-    ],
+    body("email")
+      .exists()
```

Comment:
It's not a must to supply both email and phone number because only one of the two is required for registration. cc: @NoahNsimbe